### ISSUE TITLE
The output score of an individual-level feature

### ISSUE THREAD

**Issue by JWKKWJ123** [interpretml/interpret#568]
Hi all,
I would like to ask if I want to use the ebm.eval_terms(.) to get local explanations or the ebm.term_importances(.) to get global explanations, what is the exactly of output score of an individual-level feature?  Is it exactly the output of shape functions f<sub>k</sub>(.), or the a normalized number?

**Comment by paulbkoch** [comment]
Hi @JWKKWJ123 -- eval_terms finds the vertical value on the shape plots for the feature values in X.  For EBMs, that value is equal to the local explanations.

term_importances are not normalized, and no sigmoid is applied.  term_importances can be calculated with eval_terms.  If you take the mean of the absolute values returned from eval_terms, it will be equal to the term importances.

**Comment by paulbkoch** [comment]
For classification, the scores are in logits and should be comparable across datasets.  For regression, the scores are in the units being predicted, so you'll probably want to normalize across datasets.  There isn't a generally agreed upon normalization for regression.  I use the interquartile range in our benchmarks.
